iis-wf/iis-wf-affmatching/src/test/resources/data/expectedOutput/matchedAffiliations.json

```diff
@@ -10,3 +10,15 @@
     "organizationId":"20|UW",
     "matchStrength":0.9677419
 }
+{
+    "documentId":"DOC_3",
+    "position":1,
+    "organizationId":"20|UCB",
+    "matchStrength":0.19354838
```

Comment:
Reason behind low matchStrength in this case is because affiliation was matched only by `SectionedNameStrictCountryLooseMatchVoter` and `SectionedNameLevenshteinCountryLooseMatchVoter` (third and fifth position on voters list).
I agree that we should consider some changes to pump this value up.